packages/api-client/src/views/Collection/components/MarkdownInput.vue

```diff
@@ -48,19 +48,19 @@ watch(mode, (newMode) => {
       </ScalarButton>
     </div>
     <div
-      class="has-[:focus-visible]:bg-b-1 z-1 group relative flex flex-col rounded-lg">
+      class="has-[:focus-visible]:bg-b-1 group relative z-1 flex flex-col rounded-lg">
       <div class="flex h-full min-h-[calc(1rem*4)] flex-col">
         <!-- Preview -->
         <template v-if="mode === 'preview'">
           <template v-if="modelValue && modelValue.trim().length">
             <ScalarMarkdown
               v-if="modelValue"
-              class="hover:border-b-3 h-full flex-1 rounded border border-transparent p-1.5"
```

Comment:
`border-b-3` can also mean `border-bottom: 3px` and Tailwind v4 doesn't like that